src/Umbraco.Cms.Api.Management/Mapping/Package/PackageViewModelMapDefinition.cs

```diff
@@ -0,0 +1,69 @@
+using Umbraco.Cms.Api.Common.ViewModels.Pagination;
+using Umbraco.Cms.Api.Management.ViewModels.Package;
+using Umbraco.Cms.Core.Mapping;
+using Umbraco.Cms.Core.Packaging;
+
+namespace Umbraco.Cms.Api.Management.Mapping.Package;
+
+public class PackageViewModelMapDefinition : IMapDefinition
+{
+    public void DefineMaps(IUmbracoMapper mapper)
+    {
+        mapper.Define<PackageDefinition, PackageDefinitionViewModel>(
+            (source, context) => new PackageDefinitionViewModel()
+            {
+                Name = string.Empty,
+                PackagePath = string.Empty
+            },
+            Map);
+        mapper.Define<InstalledPackage, PackageMigrationStatusViewModel>((source, context) => new PackageMigrationStatusViewModel() { PackageName = string.Empty }, Map);
+        mapper.Define<IEnumerable<PackageDefinition>, PagedViewModel<PackageDefinitionViewModel>>((source, context) => new PagedViewModel<PackageDefinitionViewModel>(), Map);
+        mapper.Define<IEnumerable<InstalledPackage>, PagedViewModel<PackageMigrationStatusViewModel>>((source, context) => new PagedViewModel<PackageMigrationStatusViewModel>(), Map);
```

Comment:
```suggestion
        mapper.Define<PackageDefinition, PackageDefinitionViewModel>(
            (_, _) => new PackageDefinitionViewModel
            {
                Name = string.Empty,
                PackagePath = string.Empty,
            },
            Map);
        mapper.Define<InstalledPackage, PackageMigrationStatusViewModel>((_, _) => new PackageMigrationStatusViewModel { PackageName = string.Empty }, Map);
        mapper.Define<IEnumerable<PackageDefinition>, PagedViewModel<PackageDefinitionViewModel>>((_, _) => new PagedViewModel<PackageDefinitionViewModel>(), Map);
        mapper.Define<IEnumerable<InstalledPackage>, PagedViewModel<PackageMigrationStatusViewModel>>((_, _) => new PagedViewModel<PackageMigrationStatusViewModel>(), Map);
```

We can discard the parameters if we don't need them 😄